core/module/included_spec.rb

```diff
@@ -36,4 +36,8 @@ def test
   it "is private in its default implementation" do
     Module.should have_private_instance_method(:included)
   end
+
+  it "works with super using a singleton class" do
+    ModuleSpecs::SingletonOnModuleCase::Bar.included_called?.should == true
```

Comment:
The actual include should be done in the spec example, otherwise correctly loading the fixtures file needs to handle that case. (Module#include is public BTW, no need for `send`)